src/wchar/mbsrtowcs_s.c

```diff
@@ -180,9 +181,8 @@ mbsrtowcs_s (size_t *restrict retval,
             dest[*retval] = L'\0';
 #endif
         }
-        return EOK;
```

Comment:
I believe the early return generates better code, than passing it through.
Is it just for consistency?

Reply:
IAR was giving an error for the unreachable return statement at line 205 (`return RCNEGATE(ESNOSPC);`). It's unreachable because both the `if` and `else` have return statements. We could just delete line 205 if you prefer.